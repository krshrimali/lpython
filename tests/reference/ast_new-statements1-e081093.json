{
    "basename": "ast_new-statements1-e081093",
    "cmd": "lpython --show-ast --new-parser --no-color {infile} -o {outfile}",
    "infile": "tests/parser/statements1.py",
    "infile_hash": "766942b29db2b91450cd50fa02e4b97d325b9037d8abbe2f2295b2c3",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "ast_new-statements1-e081093.stderr",
    "stderr_hash": "2bed91c758a98b07b72e243586b51167001df706e67dfe022953fa42",
    "returncode": 1
}