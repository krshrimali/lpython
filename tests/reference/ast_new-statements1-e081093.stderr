syntax error: Token ':' is unexpected here
   --> tests/parser/statements1.py:180:18
    |
180 | for item in list1:
    |                  ^ 
